mods/ts/chrome/ingame-player.yaml

```diff
@@ -151,39 +159,51 @@ Container@PLAYER_WIDGETS:
 					Children:
 						LogicTicker@RADAR_TICKER:
 						ColorBlock@RADAR_FADETOBLACK:
```

Comment:
You need to move this below the insignia to fix glitches like 
![image](https://cloud.githubusercontent.com/assets/756669/6765392/4979bb92-cfde-11e4-966a-3165fd83e05c.png)

See https://github.com/OpenRA/OpenRA/pull/7693.